### ISSUE TITLE
Move projects in separate repositories

### ISSUE THREAD

**Issue by bandali** [unitb/literate-unitb-complete#33]
From literate-unitb created by [aminb](https://github.com/aminb) : unitb/literate-unitb#32  

Originally reported by: **Simon Hudon (Bitbucket: [cipher2048](https://bitbucket.org/cipher2048), GitHub: Unknown)**

---
- [x] Web prover: [literate-unitb/unitb-web](/literate-unitb/unitb-web)
- [x] existential: [literate-unitb/existential](/literate-unitb/existential)
- [x] tex2png-hs: [literate-unitb/tex2png-hs](/literate-unitb/tex2png-hs)
- [x] printf-th: [literate-unitb/printf-th](/literate-unitb/printf-th)
- [x] invariants: [literate-unitb/invariants](/literate-unitb/invariants)
- [x] axiomatic-classes: [literate-unitb/axiomatic-classes](/literate-unitb/axiomatic-classes)

---
- Bitbucket: https://bitbucket.org/literateunitb/literate-unitb/issue/32

**Comment by bandali** [comment]
Done.

Next step: sort out the build issues and then remove the libraries from the main (this) repo.

**Comment by bandali** [comment]
We did this, moving every package to its own repo, we had ~30 repos or so. Turns out it wasn't the best approach and made development [more inconvenient](https://www.reddit.com/r/haskell/comments/640dzy/question_architecture_workflow_in_a_sizeable/) than it had to be, due to having to track and update the git hash of each of the dependency repos for every commit.

We've returned to the monolith repo architecture, with a couple of exceptions being libraries that could really be useful outside of Literate Unit-B. So we're sticking with separate packages, just that now they're all in one repo.
